websites/M/Magister/dist/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "QkeleQ10",
+    "id": "722738307477536778"
+  },
+  "url": "magister.net",
+  "regExp": "([a-z0-9-]+[.])*magister[.]net[/]",
+  "description": {
+    "en": "Magister 6 is a popular student information system in the Netherlands.",
+    "nl": "Magister 6 is een populair leerlingvolgsysteem in Nederland."
+  },
+  "version": "1.0.0",
+  "service": "Magister",
+  "logo": "https://i.imgur.com/1TtM9A1.png",
+  "thumbnail": "https://i.imgur.com/TVjDmC4.png",
+  "color": "#1F97F9",
+  "tags": [
+    "leerlingvolgsysteem",
+    "lvs",
+    "magister",
+    "6",
```

Comment:
6? xd

Reply:
yeah idk the program is officially called Magister 6

Reply:
but it isnt 6, delete it fr pls